source/developers-guide/plugin-configuration/index.md

```diff
@@ -173,6 +173,51 @@ Below all supported configuration elements including their design and source cod
 
 <img src="img/numberfield.png" alt="Numberfield" class="image-border" />
 
+### Boolean
+To define something similar like checkbox you can use `boolean`
+```xml
+<elements>
+    <element required="false" type="boolean" scope="shop">
+        <name>show</name>
+        <label lang="de">Anzeigen</label>
+        <label lang="en">Show</label>
+        <description lang="de">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut.</description>
+        <description lang="en">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut.</description>
+    </element>
+</elements>
+```
+Notice: Fields like `name`, `label`,`description` are not required. They just listed for code transparency.
+
+This will provide you with a prefilled dropdown.
+
+<img src="img/boolean.png" alt="Booleanfield" class="image-border" />
+
+Currently there is no element for checkbox at the moment. 
```

Comment:
please remove this part as stated here: https://github.com/shopware/devdocs/pull/808#issuecomment-363733637